architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
-xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
-container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+An _image stream_ is a mapping between tags in {product-title} objects and 
+xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker image] images. 
+You can think of an image stream as a wrapper or abstraction
+for Docker images to see what images are available and to ensure that you are using the specific image you need even if the image in 
+the repository changes.
+Image streams do not contain actual image data, but presenta single virtual view of related images, similar to an image repository. 
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+Yoyu can  configure xref:../dev_guide/builds/triggering_builds.adoc#image-change-triggers[Builds] and
+xref:../dev_guide/deployments/basic_deployment_operations.adoc#image-change-trigger[Deployments] to watch an image stream
+for notifications when new images are added and react by performing a Builds or Deployment, respectively.
+
+For example, if a Deployment is using a certain image and a new version of that image is created, a Deployment could be automatically performed.
+
+//from https://blog.openshift.com/image-streams-faq/
+This means that even if the source image changes, the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
```

Comment:
this looks out of place.  or needs more context around it.

the main point here is that if the imagestreamtag used by the deployment/build is *not* updated, then even if the docker image in the docker registry is updated, the build/deployment will continue using the previous (presumably known good) image.